redash/controllers.py

```diff
@@ -488,6 +488,22 @@ def csv_response(query_result):
         headers.update(cache_headers)
         return make_response(s.getvalue(), 200, headers)
 
+    @staticmethod
+    def add_access_control_allow_origin_header(headers={}):
```

Comment:
they should be. I think the generate make_response adds the headers you send on top of the default headers.